demo/crypto-perf/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM centos:7
+
+ENV http_proxy http://proxy.ir.intel.com:911/
+ENV https_proxy http://proxy.ir.intel.com:911/
+ENV no_proxy http://proxy.ir.intel.com:911
+ENV SHELL=/bin/bash
+COPY ./resolv.conf /etc/resolv.conf
+RUN rpm --rebuilddb
+RUN yum -y clean all; yum -y update; RUN yum -y install epel-release  httpd httpd-tools curl traceroute tcpdump bind-utils nc net-tools nmap screen htop cairo-devel hwloc-libs hwloc-gui pciutils
```

Comment:
This second `RUN` in the line will be lost.

Also it's better to have only one `RUN` for all the shell commands. This way only one overlay FS layer will be created when building the image.